### ISSUE TITLE
Actionbar text not working with effortless-fabric installed.

### ISSUE THREAD

**Issue by Encrypted-Thoughts** [huskcasaca/effortless#17]
### Issue:
With Effortless Fabric 1.6.2 installed actionbar messages seem to stop working. An easy place to see this is to just use a bed, but it also seems to affect mods that would try to use the actionbar as well by doing something like `player.sendMessage("some message", true)` which is how I first noticed it.

### Steps to Reproduce:
1. Create a new 1.19.3 instance with fabric loader 0.14.12 installed.
2. Install Effortless Fabric 1.6.2 and Fabric API 0.70.0+1.19.3
3. Create a new world and try to sleep in a bed. 

![no actionbar text](https://user-images.githubusercontent.com/50642352/210156427-a35952d7-bc26-448f-83e5-edd17336e0ed.gif)
![actionbar text](https://user-images.githubusercontent.com/50642352/210156455-88c24c65-baa9-4a75-9384-9ebec1ef594d.gif)

**Comment by Encrypted-Thoughts** [comment]
Seems to be because of the below code in BlockPreviewRender's render method constantly setting the actionbar text to an empty string.

```java
  //Expire
  if (currentPlacing.isEmpty()) {
      Effortless.log(player, "", true);
  }
```

I built a version for myself with it stripped out and haven't noticed anything being obviously broken. Can it just be removed or does it serve a purpose I haven't found yet?

**Comment by huskcasaca** [comment]
Oh already fixed in branch `dev-1.6.3` but haven't release. You can checkout and rebuild.

**Comment by huskcasaca** [comment]
https://github.com/huskcasaca/effortless-fabric/releases/tag/v1.6.3
